### PR TITLE
Package explainer

### PR DESCRIPTION
**Pull request by zhangyongjing**
Add the explainer for 'Packaging', clarifying the differences from other packaging technologies.

@QingAn @zhiqiangyu @ShourenLan @xfq

### CODE REVIEW

File: specs/packaging/docs/explainer.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# MiniApp Package explainer
+
+> Note: This document serves as a supplementary explanation of the [MiniApp Packaging](https://github.com/w3c/miniapp/blob/gh-pages/specs/packaging/index.html) spec. If there is any inconsistency with the spec, you should consider the spec to be authoritative.
```
Comment (xfq) [inline specs/packaging/docs/explainer.md:3]:
```suggestion
> Note: This document serves as a supplementary explanation of the [MiniApp Packaging](https://w3c.github.io/miniapp/specs/packaging/) spec. If there is any inconsistency with the spec, you should consider the spec to be authoritative.
```

File: specs/packaging/docs/explainer.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+A MiniApp Package is a [ZIP](https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT)-based container file that wraps a set of files (JS code, components, resources, configurations, styles) comprising the whole MiniApp. 
+It can be delivered to a hosting platform through different channels (e.g. web, app store, offline), then rendered/executed by the hosting platform.
+
+### Why should we care?
+
+A MiniApp Package contains all necessary information used by a hosting platform to check, load and execute the MiniApp properly. A standardized format is necessary so that a MiniApp package generated by different developers or tools can be deployed on different hosting platforms (e.g. super-apps, or native-os frameworks) in a compatible and consistent way. The spec shall define the following aspects to ensure interoperability:
+* minimum set of files for a MiniApp to be runnable
+* optional files for enhanced features
+* a well-defined folder structure
```
Comment (xfq) [inline specs/packaging/docs/explainer.md:21]:
```suggestion
* a well-defined file and directory structure
```

File: specs/packaging/docs/explainer.md (hunk 1 above)
Comment (zhangyongjing) [reply specs/packaging/docs/explainer.md:3]: accepted

File: specs/packaging/docs/explainer.md (hunk 2 above)
Comment (zhangyongjing) [reply specs/packaging/docs/explainer.md:21]: accepted

**Pull request merged**